engine/consensus/dkg/messaging_engine.go

```diff
@@ -13,6 +16,13 @@ import (
 	"github.com/onflow/flow-go/network"
 )
 
+// RETRY_MAX is the maximum number of times the engine will attempt to forward
+// a message
+const RETRY_MAX = 5
+
+// RETRY_MILLISECONDS is the number of milliseconds to wait between the two first tries
+const RETRY_MILLISECONDS = 1000
```

Comment:
nitpicking, but usually in go we use duration like `1000*time.Millisecond`, improves readability

Reply:
@jordanschalm ^

Reply:
My comment is about the naming and the constant visibility: `retryMilliseconds` maybe?

Reply:
I agree we should use the `Duration` type, my point was _given_ we are using the more abstract `Duration` type (which abstracts away the scale and unit of measure) rather than a numeric type, we should not include a specific unit or scale of measure in the variable name.